src/typeahead/typeahead-window.ts

```diff
@@ -4,8 +4,9 @@ import {NgbHighlight} from './highlight';
 @Component({
   selector: 'ngb-typeahead-window',
   exportAs: 'ngbTypeaheadWindow',
+  host: {'class': 'dropdown-menu', 'style': 'display: block'},
   template: `
-    <div class="dropdown-menu" aria-labelledby="dropdownMenu1">
+    <div aria-labelledby="dropdownMenu1">
```

Comment:
Yes, better to move everything I guess. It was because of the positioning. Positioning is applied to the host, and it's because of WIP :)